server/plugin/command.go

```diff
@@ -221,6 +221,31 @@ func (p *Plugin) handleSubscriptions(c *plugin.Context, args *model.CommandArgs,
 	}
 }
 
+func (p *Plugin) handleCreate(_ *plugin.Context, args *model.CommandArgs, parameters []string, _ Client, _ bool) string {
+	if len(parameters) == 0 {
+		return "Invalid create command. Available commands are 'incident' and 'request'."
```

Comment:
Is the `request` subcommand also backported to v2?

Reply:
no, updated this